### PR TITLE
refactor: refactor some places

### PR DESCRIPTION
**Pull request by arvgord**
- Configured automatic logging configuration from lo4j2.xml file
- Added code to get single DICOM Store
- Moved arguments description to cli-messages.properties file
- Fixed typos

### CODE REVIEW

File: src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java [hunk 1]
```diff
@@ -174,10 +185,10 @@ public Series getSingleSeries(QueryBuilder queryBuilder) throws DicomFuseExcepti
...
         .addParameter(PARAM_SERIES_ID, queryBuilder.getSeriesId())
         .setPath(path);
-    List<Series> series = createRequestForObjectList(uriBuilder, new TypeReference<List<Series>>() {
-    });
-    if (series.size() < 1) {
-      throw new DicomFuseException("Series not found");
+    List<Series> series =
+        createRequestForObjectList(uriBuilder, new TypeReference<List<Series>>() {});
+    if (series.size() == 0) {
+      throw new DicomFuseException("Series not found!");
```
Comment (dzelemba) [inline src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java]: Let's avoid adding exclamation marks to error messages. It's already an error, no need to emphasize more.

File: src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java [hunk 2]
```diff
@@ -210,25 +210,33 @@ public void setInstanceCommand(DicomPath dicomPath, Command command) throws Dico
   }
 
   public boolean isDatasetOutdated() {
-    return cachedDataset.getDatasetCacheTime().isBefore(Instant.now());
+    Instant instantNow = Instant.now();
+    return cachedDataset.getDatasetCacheTime().isBefore(instantNow) ||
+        cachedDataset.getDatasetCacheTime().equals(instantNow);
```
Comment (dzelemba) [inline src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java]: Why did you have to make this change?

File: src/main/java/com/google/dicomwebfuse/fuse/DicomPathParser.java [hunk 3]
```diff
@@ -62,13 +64,21 @@ DicomPath parsePath(String path, Command command) throws DicomFuseException {
         if (tempDicomPath != null) {
           dicomPath = tempDicomPath;
         } else {
+          // UIDs should contain only numbers
```
Comment (dzelemba) [inline src/main/java/com/google/dicomwebfuse/fuse/DicomPathParser.java]: Unit test for these changes?

File: src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java (hunk 2 above)
Comment (dzelemba) [reply src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java]: Should there be unit tests covering this?

File: src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java (hunk 1 above)
Comment (arvgord) [reply src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java]: Ok, I understood. Done

File: src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java (hunk 2 above)
Comment (arvgord) [reply src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java]: > Why did you have to make this change?

Instant.now()  - Returns a value in microseconds in Java 9 and later, but is restricted to mere milliseconds in Java 8. That's why in some cases during a unit test(DicomFuseHelperTest) I got heisenbug. Sometimes tests were failed sometimes were successful. I found that code may return false if instantNow is less than 1 millisecond(if code faster than 1 millisecond).

> Should there be unit tests covering this?

This was done in the DicomFuseHelperTest. If the tests are successful it means that this code works.

**Comment by arvgord** [comment]
Should we change the patch version to every PR?

File: src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java (hunk 2 above)
Comment (dzelemba) [reply src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java]: Ok please add a code comment explaining this.

**Comment by dzelemba** [comment]
No we don't need to update the patch version each time, I'll just manually update it when we release.

File: src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java (hunk 2 above)
Comment (arvgord) [reply src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java]: Done

File: src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java [hunk 4]
```diff
@@ -205,30 +205,77 @@ public void setInstanceCommand(DicomPath dicomPath, Command command) throws Dico
...
       default:
-        throw new DicomFuseException("Invalid dicom path level! " + dicomPath);
+        throw new DicomFuseException("Invalid dicom path level - " + dicomPath);
     }
   }
 
+  /**
+   * Checks that Dataset outdated or not.
+   *
+   * {@code Instant.now()} - Returns a value in microseconds in Java 9 and later, but returns
```
Comment (dzelemba) [inline src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java]: This shouldn't be part of the Javadoc of this method... a user using this method doesn't need to know these details. Just put the comment inside the function definition.

Comment (arvgord) [reply src/main/java/com/google/dicomwebfuse/entities/cache/Cache.java]: Done

**Pull request merged**